code/game/objects/machinery/Sleeper.dm

```diff
@@ -445,34 +445,46 @@
 		beaker.loc = usr.loc
 		beaker = null
 
-
-/obj/machinery/sleeper/verb/move_inside()
-	set name = "Enter Sleeper"
-	set category = "Object"
-	set src in oview(1)
-
-	if(usr.stat || !ishuman(usr))
+/obj/machinery/sleeper/relaymove(mob/user)
+	if(user.incapacitated(TRUE)) 
 		return
+	go_out()
 
-	var/mob/living/carbon/human/user = usr
+/obj/machinery/sleeper/proc/move_inside_wrapper(mob/M, mob/user)
```

Comment:
kinda nitpicking but this is the opposite of a wrapper, do_move_inside is a more apt name.